refinery/data_set_manager/test_views.py

```diff
@@ -861,9 +861,9 @@ def test_get_returns_derived_nodes_for_dataset_with_related_nodes(self):
         force_authenticate(get_request, user=self.user)
         get_response = self.view(get_request, file_node.uuid)
         self.assertEqual(get_response.status_code, 200)
-
-        derived_nodes_uuid = [node.uuid for node in Node.objects.filter(
-            assay=file_node.assay) if node.is_derived()]
+        assay_nodes = Node.objects.filter(assay=file_node.assay)
```

Comment:
You could think about updating the query with an `exclude()` to make the list comprehension below simpler.
But this isn't crucial

```suggestion
        assay_nodes = Node.objects.filter(assay=file_node.assay).exclude(id=file_node.id)
        derived_nodes_uuid = [node.uuid for node in assay_nodes if node.is_derived()]
```